src/utils/BrambleExtensionLoader.js

```diff
@@ -34,6 +34,7 @@ define(function (require, exports, module) {
         "bramble",
         "Autosave",
         "brackets-paste-and-indent",
+        "brackets-show-whitespace",
```

Comment:
I have serious misgivings about making this a default that we load.  @flukeout this needs to be tested by you, and I also want to do performance testing to make sure it doesn't bog down the editor.